File: Quick/1.dart
Wap to print all negative elements in an array.

import 'dart:io';

void main() {
  List elements = [];
  int num;
  int n;

  stdout.write("Enter the number : ");
  n = int.parse(stdin.readLineSync()!);

  for (int i = 0; i < n; i++) {
    stdout.write("Enter value of Element[$i] : ");
    num = int.parse(stdin.readLineSync()!);
    elements.add(num);
  }

  for (var i = 0; i < n; i++) {
    if (elements[i] < 0) {
      print(elements[i]);
    }
  }
}

File: 1.c
#include<stdio.h>
#include<conio.h>

void main()
{
    int n;

    printf("Enter the number : ");
    scanf("%d", &n);

    int num = 1;
    int y = 1;

    for (int  i = 1; i <= n; i++)
    {
        for (int j = 1; j < i; j++)
        {
            printf("-  ");
        }
        
        for (int j = i; j <= n; j++)
        {
           printf("%d ",num);
           if (num >= 1 && num <= 9)
           {
            printf(" ");
           }
            num = num + j;
        }

        for (int j = 9; j >= i; j--)
        {
            printf("%d ",num);
            num = num + j;
        }
        
        y += i;
        num = y + i;
        printf("\n");
    }
    
}